tests/test_repl.py

```diff
@@ -0,0 +1,10 @@
+from unittest import mock
+
+
+@mock.patch("snowflake.cli._plugins.sql.manager.SqlExecutionMixin._execute_string")
+def test_repl_input_handling(mock_execute, runner, mock_cursor, snapshot):
+    mock_execute.return_value = (mock_cursor(["row"], []) for _ in range(1))
+    query = "select 1;"
+    result = runner.invoke(["sql"], input=query)
+    assert result.exit_code == 0
+    snapshot.assert_match(result.output)
```

Comment:
I don't see `select 1` in the snapshot, are we sure the test is working as expected?